### DISCUSSION TITLE
Card doesn’t exist on website

### DISCUSSION THREAD

**Discussion by VladislavAntonyuk** [WeAreDotnet/dotnet-creator-cards#162] [Q&A]
I have created json files and they are merged, however I still don’t see my card in the website. Also my card doesn’t exist in the creators list. Do I need manually fill information on the website as well?

**Comment by DeeJayTC** [comment]
Here's  your profile:

https://www.wearedotnet.io/VladislavAntonyuk

I can see its not loading your feed properly, let me find out why

**Reply by DeeJayTC** [reply]
Apparently the RSS Serializer either can't find or does not support the RSS version your feed is using. 
Can you tell me what exactly you're using?

**Reply by DeeJayTC** [reply]
Yea so long story short @VladislavAntonyuk 
There's two  major issues with your feed:

* The PubDate is not in a proper format, has to be an ISO or RFC-compliant timestamp
* Your RSS Feed has to be at RSS 2.0 or ATOM

**Reply by VladislavAntonyuk** [reply]
Thank you. These 2 issues are fixed

**Comment by DeeJayTC** [comment]
Also, your card exists as well :)

![image](https://user-images.githubusercontent.com/4077759/235189964-8f50d05f-dc41-4dcd-a7a9-9dc2bdbf12e9.png)

**Reply by VladislavAntonyuk** [reply]
I tried to search by "vlad" and didn't find any card.
Also, could you please explain what is the difference between these 3 files
- WeAreDotnet/dotnet-creator-cards/planet-xamarin-creator-cards/destination/vladislav-antonyuk.json
- WeAreDotnet/dotnet-creator-cards/planet-xamarin-creator-cards/source/VladislavAntonyuk.json
- WeAreDotnet/dotnet-creator-cards/creator-cards/vladislav-antonyuk.json

When I logged in to the website using my GitHub account, my profile is empty:
![image](https://user-images.githubusercontent.com/33021114/236419067-ddf79092-2e7a-425e-ba00-64ae8830286c.png)

**Reply by DeeJayTC** [reply]
Ahh! We took over some data from PlanetXamarin and apparently missed that you already had an account. 

I merge those
